libraries/plugins/p2p/include/steem/plugins/p2p/p2p_default_seeds.hpp

```diff
@@ -28,22 +28,27 @@ const std::vector< std::string > default_seeds = {
    "seed.roelandp.nl:2001",               // roelandp
    "steem.global:2001",                   // klye
    "seed.esteem.ws:2001",                 // good-karma
-   "94.23.33.61:2001",                    // timcliff
+   "seed.timcliff.com:2001",              // timcliff
    "104.199.118.92:2001",                 // clayop
-   "192.99.4.226:2001",                   // dele-puppy
-   "seed.bhuz.info:2001",                 // bhuz
    "seed.steemviz.com:2001",              // ausbitbank
    "steem-seed.lukestokes.info:2001",     // lukestokes
-   "seed.blackrift.net:2001",             // drakos
+   "seed.steemian.info:2001",             // drakos
    "seed.followbtcnews.com:2001",         // followbtcnews
-   "node.mahdiyari.info:2001",            // mahdi.yari
+   "node.mahdiyari.info:2001",            // mahdiyari
    "seed.jerrybanfield.com:2001",         // jerrybanfield
```

Comment:
node is down (and was down at last three times on multiple recurring checks during last week)